draft-ma-opsawg-schedule-yang.md

```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
    The "schedule-id" parameter is useful to uniquely identify a schedule in
    a network device or controller if multiple scheduling contexts exists.
 
-   The "state" parameter is defined to configure/expose the sheduling state, depends
-   on the use of the grouping. Likewise, the "version" parameter summarizes the
-   current schedule version information.
+   The "state" parameter is defined to configure/expose the sheduling state,
+   depending on the use of the grouping. The "identityref" type is used for this
+   parameter to allow extensibility in future modules. For example, a "conflicted"
+   state is valid in scheduling contexts where multiple systems struggle for the
+   scheduling of the same property.
 
-   The "last-update" parameter identifies when the schedule was last modified.
+   The "version" parameter summarizes the current schedule version information,
+   and the "last-update" parameter identifies when the schedule was last modified.
```

Comment:
```suggestion
   The "last-update" parameter identifies when the schedule was last modified. In some contexts, this parameter can be
   used to track the configuration of a given schedule. In such cases, the "version" is not used.
```

Reply:
> In such cases, the "version" is not used.

In such cases, the "version" may not be used. 
Better? i think these two parameters are  not conflicting. An iteration version number helps obtain the number of historical modifications, while last-update can be used to understand when it was last modified.

Reply:
Ok for "may not be" for now. Edited the change  accordingly. 

I expect we will have this comment during the publication process to clarify the relationship between the two.

Reply:
Okay, sure, I am okay to drop "version" parameter if there is any objection in the future.